### DISCUSSION TITLE
Бронежилеты и их бронепластины

### DISCUSSION THREAD

**Discussion by sobakka265** [octothorp-team/public-dobrograd#901] [Предложения]
Здрасте товарищи краймеры и другие любители наводить суету в прекрасном городе Доброград, я бы хотел предложить идею по улучшению бронежилетов, ибо пониженный урон это конечно круто, но хотелось бы видеть полноценную механику брони по типу полного поглощения урона в грудь.( я хз какой класс зашиты у броников) И вот что мне хотелось бы видеть, лёгкий может с определённым процентом выдержать попадания с пистолета, средний будет иметь повышенный шанс защиты, но он также сможет помочь против ПП, тяжёлый всё тоже самое с повышенном шансов+шанс не пробития тяжёлым патроном.

**Comment by Kyper1** [comment]
У нас и так пули в голое тело не регают, смысл от поглощения?

**Reply by sobakka265** [reply]
ну тут не ко мне вопрос, а к технической части проекта(говно драное). Так то я согласен но, может вдруг пули наконец то будет регаться по человечески и тогда это будет полезно
